README.md

```diff
@@ -1,6 +1,6 @@
 # 2048FX
 
-This is project is forked from https://github.com/brunoborges/fx2048
+This project is forked from https://github.com/brunoborges/fx2048
```

Comment:
For this PR, you may want to remove this line :-)

Reply:
Sure - yet, this PR is not meant to be merged. I dropped a lot features, besides Gradle.